README.md

```diff
@@ -11,27 +11,31 @@ Want to contribute? Do the following:
 1. `git clone` this repository to your local computer, or set up a GitHub
    codespace to the same effect.
 
-2. Set up your virtual environment and install the requirements. I'm running
+1. Set up your virtual environment and install the requirements. I'm running
    this on Python 3.13.1.
 
-  - Initialize the virtual environment:
-```
+Initialize the virtual environment:
```

Comment:
These commands are slightly different for Windows. Do we want to add instructions on how to create a virtual environment for different platforms, or trust that ppl can search/chatGPT their way through?

Reply:
Trust ppl for now; there won't be that many that stumble here.